src/open_inwoner/utils/decorators.py

```diff
@@ -72,9 +76,14 @@ def cache(
     of the wrapped function as interpolation placeholders. When deocarating a method,
     you can also include instance attributes using the `"cache:{self.attr}"` syntax.
     :param alias: the Django cache to use, defaults to "default"
-    :param timeout: the timeout for the cache in seconds. Defaults to 60
+    :param timeout: the timeout for the cache in seconds. Defaults to 60. Note that you
+    cannot cache forever (`None` will be interpreted as a request to use the default),
+    but you can set this to `0` to avoid caching, as per the Django low-level cache
+    docs.
     """
 
+    _timeout = timeout if timeout is not None else _DEFAULT_CACHE_TIMEOUT
```

Comment:
I agree that we should disallow caching forever (though I'm not sure why anyone would want to try that), but the approach to `None` seems needlessly complicated and potentially confusing. Instead of re-interpreting some expected behavior in a different way because that is found more desirable (`timeout=None` doesn't mean that `timeout` is `None`, as one would expect, but rather `timeout` is `default` because that is found more desirable), I would prefer to keep the meaning of the original behavior and simply disallow it. That is, `if not isinstance(timeout, int): raise ValueError` or something like it.

What is the advantage of this approach to `None`? Perhaps I'm missing something...